lib/screens/repository_screen/repository_screen.dart

```diff
@@ -0,0 +1,57 @@
+import 'package:flutter/material.dart';
+import 'package:top_git_rep/models/search_result_item.dart';
+
+class RepositoryScreen extends StatelessWidget {
+  const RepositoryScreen({Key? key, required this.repository})
+      : super(key: key);
+
+  final List<Repository> repository;
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+        appBar: AppBar(
+            title: const Text('Repositories'),
+            leading: GestureDetector(
+              child: IconButton(
+                icon: const Icon(Icons.arrow_back),
+                onPressed: () {
+                  // context.read<GithubSearchBloc>().add(InitEvent());
+                  Navigator.pop(context);
+                },
+              ),
+            )),
+        // ),
+        body: ListView.builder(
+            itemCount: repository.length,
+            itemBuilder: (BuildContext context, int index) {
+              return Padding(
+                padding: const EdgeInsets.all(3),
```

Comment:
Add all dimensions from project to constants.For example, you can make `Dimensions` class with static fields.